src/containers/FormikForm/formikDraftHooks.js

```diff
@@ -39,25 +40,51 @@ export function useFetchArticleData(articleId, locale) {
         ...e,
         articleType: 'concept',
       }));
+
+      const [relatedArticles, externalLinks] = partition(
+        article.relatedContent,
+        e => typeof e === 'number',
+      );
+
+      const convertedRelatedArticles = await fetchArticleList(relatedArticles);
+
+      const convertedRelatedContent = convertedRelatedArticles.concat(externalLinks);
+
       const taxonomy = await fetchTaxonomy(articleId, locale);
       setArticle(
-        transformArticleFromApiVersion({ taxonomy, ...article }, locale, convertedConcepts),
+        transformArticleFromApiVersion(
+          { taxonomy, ...article },
+          locale,
+          convertedConcepts,
+          convertedRelatedContent,
+        ),
       );
       setLoading(false);
     }
   };
 
+  const fetchArticleList = async articleIds => {
+    return Promise.all(
+      articleIds.map(async elementId => {
+        return await draftApi.fetchDraft(elementId);
```

Comment:
```suggestion
        return draftApi.fetchDraft(elementId);
```
Pirk. Kan også plasseres på én linje